yarn.lock

```diff
@@ -3236,6 +3236,11 @@ ansi-regex@^2.0.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-2.1.1.tgz#c3b33ab5ee360d86e0e628f0468ae7ef27d654df"
   integrity sha1-w7M6te42DYbg5ijwRorn7yfWVN8=
 
+ansi-regex@^3.0.0:
```

Comment:
i often see this dep popping up, is it because of different node js version?

Reply:
```
node -v
v12.16.0
yarn -v 
1.22.0
```

Reply:
We've done our researched with @marie-j for the last two weeks and we weren't able to a culprit yet